server/src/services/media.service.ts

```diff
@@ -214,7 +214,8 @@ export class MediaService extends BaseService {
       const colorspace = this.isSRGB(asset) ? Colorspace.SRGB : image.colorspace;
       const processInvalidImages = process.env.IMMICH_PROCESS_INVALID_IMAGES === 'true';
 
-      const decodeOptions = { colorspace, processInvalidImages, size: image.preview.size };
+      const orientation = Number(asset.exifInfo?.orientation);
```

Comment:
Prefer making it undefined instead of NaN when `asset.exifInfo?.orientation` is undefined.